.github/setup/azure/accessTokenLifeTime.sh

```diff
@@ -0,0 +1,30 @@
+
+#!/bin/bash
```

Comment:
Nitpick, bash is not always at `/bin/bash` especially on Macs, that's why I suggested `#!/usr/bin/env bash`. The shebang should also be the first line, without any blank lines above it.